addons/sale/models/sale_order.py

```diff
@@ -262,6 +262,8 @@ def _rec_names_search(self):
         comodel_name='crm.tag',
         relation='sale_order_tag_rel', column1='order_id', column2='tag_id',
         string="Tags")
+    country_id = fields.Many2one('res.country', related='partner_id.country_id', store=True, compute_sudo=True)
+    industry_id = fields.Many2one('res.partner.industry', related='partner_id.industry_id', store=True)
```

Comment:
```suggestion
    country_id = fields.Many2one(related='partner_id.country_id', store=True, compute_sudo=True)
    industry_id = fields.Many2one(related='partner_id.industry_id', store=True)
```

Also, those fields should be in the `partner-based computes` section.